src/app/Layouts/defaultPageWrapper.test.jsx

```diff
@@ -30,6 +30,14 @@ describe('defaultPageWrapper', () => {
     },
   };
 
+  window.matchMedia = jest.fn().mockImplementation(() => {
```

Comment:
I think we can skip this since we won't need it once we pull in the `aria-hidden` changes.

Reply:
Good point, removed it